ci-operator/step-registry/assisted/common/verify/assisted-common-verify-commands.sh

```diff
@@ -36,6 +36,11 @@ esac
 export KUBECONFIG=${SHARED_DIR}/kubeconfig
 mkdir -p ${ARTIFACT_DIR}/reports
 
+# copy openshift-tests binary
+podman create --name openshift-tests ${OPENSHIFT_TESTS_IMAGE}
+podman cp openshift-tests:/usr/bin/openshift-tests /usr/local/bin
+podman cp openshift-tests:/usr/bin/kubectl /usr/local/bin
```

Comment:
```suggestion
alias openshift-tests=podman run -it ${OPENSHIFT_TESTS_IMAGE} -v ${ARTIFACT_DIR}:${ARTIFACT_DIR} /usr/bin/openshift-tests
```

let's run openshift-test from its own container to avoid glibc issues like we had before.